### PR TITLE
Testing/setup cypress

### PR DESCRIPTION
**Pull request by richardltyler**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- creates boilerplate testing files 
- adds dummy data contact info `fixture` for future stubbing

## *Instructions for reviewer:*
*Where should the reviewer start?*
- run `npm run cypress` in cli
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
- some of these tests exist for future iterations and are commented out so as not to disrupt the CI/CD 

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

**Review by kristenmb — APPROVED** [review]
This looks great! I have been tinkering in the styling and changing around some class names, i'm hoping to be done with my styling branch by early afternoon. Hopefully this wont affect the tests you've already written but if it does it should be easy to refactor them.

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,18 @@
...
+before_script:
+  - npm start --silent &
 script:
-  - npm start &
   - npm run start:ci &
   - cypress run --record --key 86ce623b-9bbc-4993-8dca-f3535c528333
-deploy:
-  provider: vercel
-  app: curatour
-  skip_cleanup: true
```
Comment (DrewBradley) [inline .travis.yml:27]: I think this might have been the main issue.

**Review by DrewBradley — APPROVED** [review]
Tests are all passing in Travis now.

File: .travis.yml (hunk 1 above)
Comment (kristenmb) [reply .travis.yml:27]: Thanks for working through that!

**Pull request merged**